content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -343,7 +343,7 @@ kubectl delete svc/fine-pod
 
 Most container runtimes provide a sane set of default syscalls that are allowed
 or not. The defaults can easily be applied in Kubernetes by using the
-`runtime/default` annotation or setting the seccomp type in the security context
+`RuntimeDefault` annotation or setting the seccomp type in the security context
```

Comment:
In this case "runtime/default" was correct as the value for the annotation needed in older Kubernetes versions that lack `spec.securityContext.seccompProfile.type`